builtin/providers/profitbricks/data_source_datacenter_test.go

```diff
@@ -0,0 +1,31 @@
+package profitbricks
+
+import (
+	"github.com/hashicorp/terraform/helper/resource"
+	"regexp"
+	"testing"
+)
+
+func TestAccDataSourceDatacenter_basic(t *testing.T) {
```

Comment:
In the test, have 2 steps, the first as a setup to create the datacenter, then the second will include a datacenter and a lookup

have a look at this test as an example https://github.com/hashicorp/terraform/blob/master/builtin/providers/aws/data_source_aws_autoscaling_groups_test.go#L15